spec/features/01_visitors_sees_list_of_genres_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature "visitor sees a list of genres" do
+  scenario "Sees a list of genres and links for genres" do
+    rock = Genre.create(name:'Rock')
+
+    visit genres_path
+
+    expect(page).to have_content "Rock"
+
+  end
+  scenario "clicks link and is taken to show page for given genre" do
+    rock = Genre.create(name:'Rock')
+
+    visit genres_path
+
+    click_link "Rock"
+
+    expect(page).to have_content rock.name
+  end
+end
```

Comment:
For testing: this test should create two genres, and make expect statements ON the index page. It looks like your expect statements are actually happening on a show page.

Additionally, you should be using the `let` keyword for your object creation